src/tests/java/picard/sam/markduplicates/util/OpticalDuplicateFinderTest.java

```diff
@@ -1,13 +1,10 @@
 package picard.sam.markduplicates.util;
 
-import htsjdk.samtools.util.CollectionUtil;
 import htsjdk.samtools.util.Log;
-import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 import org.testng.Assert;
 import picard.sam.util.PhysicalLocation;
 import picard.sam.util.PhysicalLocationInt;
-import picard.sam.util.PhysicalLocationShort;
```

Comment:
This was all that remained from a big change where I tested the new version with a KDTree...but then I realized that the logging itself solved the problem...while the kdtree is much faster for the crazy edge case that I'm solving here, for most cases it is not...so I decided against using KDtree for faster analysis of an edge case that happened once in 5 years!